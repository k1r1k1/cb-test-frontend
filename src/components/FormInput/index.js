import React from 'react'

const FormInput = ({ formLabel, name, value, handleChange, error }) => {
  return (
    <div className="mt-3">
      <label className="form-label">{formLabel}</label>
      <input
        type="text"
        className={`form-control ${error ? 'is-invalid' : ''}`}
        value={value}
        onChange={handleChange}
        name={name}
      />
      {error ? <label className="invalid-feedback">{error}</label> : null}
    </div>
  )
}

export default FormInput
