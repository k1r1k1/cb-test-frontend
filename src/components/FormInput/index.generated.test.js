import renderer from 'react-test-renderer';
import React from 'react'
import FormInput from './index';

const renderTree = tree => renderer.create(tree);
describe('<FormInput>', () => {
  it('should render component', () => {
    expect(renderTree(<FormInput 
    />).toJSON()).toMatchSnapshot();
  });
  
});