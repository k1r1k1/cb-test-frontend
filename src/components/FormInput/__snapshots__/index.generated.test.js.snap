// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormInput> should render component 1`] = `
<div
  className="mt-3"
>
  <label
    className="form-label"
  />
  <input
    className="form-control "
    type="text"
  />
</div>
`;
